findAndMatch.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+
+<head>
+  <title>Complete JavaScript Course</title>
+  <script defer src="https://use.fontawesome.com/releases/v5.0.10/js/all.js" integrity="sha384-slN8GvtUJGnv6ca26v8EzVaR9DC58QEwsIk9q1QXdCU8Yu8ck/tL/5szYlBbqmS+"

```

Comment:
Note that is better to use local files for assets. It is ok for testing/prototyping new stuff to use external resources. But generally you want to have total control about what you serve to the client.